### PR TITLE
Add Security to GetAll call so now it only send back users recipes. A…

### PR DESCRIPTION
**Pull request by emritcey**
…lso, add framework for auth tokens

### CODE REVIEW

File: app/src/Pages/Recipes/Components/DisplayDBRecipeListComponent.js [hunk 1]
```diff
@@ -22,39 +24,39 @@ const useStyles = makeStyles(theme => ({
...
-    },
-    submit: {
-        margin: theme.spacing(3, 0, 2),
-    },
     head: {
-        backgroundColor: "#00bde5",
+        backgroundColor: theme.palette.primary.main,
         color: "#FFFFFF",
     },
 }));
```
Comment (Mulderink1) [inline app/src/Pages/Recipes/Components/DisplayDBRecipeListComponent.js:32]: Nice Clean-Up

File: app/src/Pages/Recipes/Components/DisplayDBRecipeListComponent.js [hunk 2]
```diff
@@ -102,7 +103,8 @@ export default () => {
     useEffect(() => {
         fetchRecipes()
         .then(body => {
-            setRecipes(body.Items);
+            console.log(body);
```
Comment (Mulderink1) [inline app/src/Pages/Recipes/Components/DisplayDBRecipeListComponent.js]: Do we need this console.log here?

File: node_app/Middleware/Recipes/GetAllRecipes.js [hunk 3]
```diff
@@ -6,11 +6,15 @@ AWS.config.update(keys.awsConfig);
...
+
     const params = {
-        TableName: "Recipe_List"
+        TableName: "Recipe_List",
+        FilterExpression : 'created_by = :created_by',
+        ExpressionAttributeValues : {':created_by' : req.params.id}
     };
 
     docClient.scan(params, (err, data) => {
+        console.log(data, err);
```
Comment (Mulderink1) [inline node_app/Middleware/Recipes/GetAllRecipes.js]: Can we take this console.log out?

**Review by Mulderink1 — APPROVED** [review]

File: app/src/Pages/Recipes/Components/DisplayDBRecipeListComponent.js (hunk 2 above)
Comment (emritcey) [reply app/src/Pages/Recipes/Components/DisplayDBRecipeListComponent.js]: good catch

**Pull request merged**